src/object/cli_obj.c

```diff
@@ -1777,8 +1774,11 @@ obj_retry_cb(tse_task_t *task, struct dc_object *obj,
 			}
 		}
 
-		delay = dc_obj_retry_delay(task, result, &obj_auxi->retry_cnt,
-					   &obj_auxi->inprogress_cnt, obj_auxi->max_delay);
+		if (!pmap_stale) {
```

Comment:
@jolivier23 I tend to use this check filter out the case of pool map refresh; would this work?

Reply:
yeah, that makes sense